docs/src/index.md

```diff
@@ -82,7 +82,6 @@ See the [Models](@ref), the [Tools](@ref tools-section), the [Tutorial](@ref), o
    [`ForwardDiff`](https://github.com/JuliaDiff/ForwardDiff.jl) to compute the
    derivatives. It has a very simple interface, though it isn't very efficient
    for larger problems.
- - [`SimpleNLPModel`](@ref): Only uses user defined functions.
```

Comment:
I can comment on the code, but I'm currently swamped. It'll have to wait until tomorrow.